SingularityBase/src/main/java/com/hubspot/singularity/SingularityRequest.java

```diff
@@ -200,7 +200,7 @@ public String getId() {
     return scheduleTimeZone;
   }
 
-  @Schema(nullable = true, description = "A count of tasks to run for long-running requests")
+  @Schema(nullable = true, description = "A count limit of tasks to run for on-demand requests")
```

Comment:
this should be an addition, not a replacement. It functions as both, depending on the context of the request type. eg. `A count of tasks to run for long-running requests or the limit on the number of concurrent tasks for on-demand requests`